03-pipefilter.md

```diff
@@ -16,46 +16,53 @@ Now that we know a few basic commands,
 we can finally look at the shell's most powerful feature:
 the ease with which it lets us combine existing programs in new ways.
 We'll start with a directory called `molecules`
-that contains six files describing some simple organic molecules.
-The `.pdb` extension indicates that these files are in Protein Data Bank format,
-a simple text format that specifies the type and position of each atom in the molecule.
+so lets go into that directory and take a look at that directory:
 
 ~~~ {.bash}
-$ ls molecules
+$ cd molecules
+~~~
+~~~ {.bash}
+$ ls
 ~~~
 ~~~ {.output}
```

Comment:
When the terminal is wide enough, a simple `ls` will in fact result in two or more columns, as in the original file. Maybe keep that as it is the behavior the learners will experience?